src/core_read.cpp

```diff
@@ -34,10 +34,9 @@ CScript ParseScript(const std::string& s)
             if (op < OP_NOP && op != OP_RESERVED)
                 continue;
 
-            const char* name = GetOpName(static_cast<opcodetype>(op));
-            if (strcmp(name, "OP_UNKNOWN") == 0)
+            std::string strName = GetOpName(static_cast<opcodetype>(op));
```

Comment:
`#include <string>` (it's not included but should have been before anyhow)

Reply:
addressed